build_tools/cmake/iree_cc_test.cmake

```diff
@@ -175,6 +175,12 @@ function(iree_cc_test)
     iree_add_test_environment_properties(${_NAME_PATH})
   endif(ANDROID)
 
+  if (NOT DEFINED _RULE_TIMEOUT)
+    set(_RULE_TIMEOUT 60)
+  endif()
+  
+
   list(APPEND _RULE_LABELS "${_PACKAGE_PATH}")
   set_property(TEST ${_NAME_PATH} PROPERTY LABELS "${_RULE_LABELS}")
+  set_property(TEST ${_NAME_PATH} PROPERTY TIMEOUT ${_RULE_TIMEOUT})
```

Comment:
Oh, this is neat: https://cmake.org/cmake/help/latest/prop_test/TIMEOUT_AFTER_MATCH.html
(I think just `TIMEOUT` is what we want, but it's good to know that if we have some setup code we don't want counted against the timeout we can use that property instead)